test/image.test.js

```diff
@@ -106,6 +106,14 @@ describe('Image', function () {
     })
   })
 
+  it('returns raw image data', function() {
+      var img = new Image();
+      img.src = png_clock;
+      var buf = img.rawData;
+      assert.ok(buf instanceof Buffer);
+      assert.strictEqual(409600, img.rawData.length);
```

Comment:
Could you swap the parameters here? So that it's `assert.strictEqual(actual, expected)`, that way the error message will be correct